### PR TITLE
Remove docs dir

### PR DESCRIPTION
**Pull request by underrun**
outdated docs removal

### CODE REVIEW

**Review by alexshtin — APPROVED** [review]
I assume you persisted the info somewhere else.

**Comment by underrun** [comment]
@cullywakelin assured me he backed up the docs in case we need them - but this is also git :-) 

@alexshtin I also updated the README to remove a ref to docs/ and adjusted some of the wording if you'd like to take another look

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Try out [Temporal Web UI](https://github.com/temporalio/web) to view your workfl
 
 We'd love your help in making Temporal great. Please review our [contribution guide](CONTRIBUTING.md).
 
-If you'd like to propose a new feature, first join the Temporal [Slack channel](https://join.slack.com/t/temporalio/shared_invite/zt-c1e99p8g-beF7~ZZW2HP6gGStXD8Nuw) to start a discussion and check if there are existing design discussions. Also peruse our [design docs](docs/design/index.md) in case a feature has been designed but not yet implemented. Once you're sure the proposal is not covered elsewhere, please follow our [proposal instructions](PROPOSALS.md).
+If you'd like to propose a new feature, first peruse our [proposals repo](https://github.com/temporalio/proposals) to discover existing active and accepted proposals. Next, join the Temporal [Slack channel](https://join.slack.com/t/temporalio/shared_invite/zt-c1e99p8g-beF7~ZZW2HP6gGStXD8Nuw) to start a discussion or check if feature has already been discussed. Once you're sure the proposal is not covered elsewhere, please follow our [proposal instructions](https://github.com/temporalio/proposals#creating-a-new-proposal).
```
Comment (cullywakelin) [inline README.md:35]: Maybe also add that new feature requests can be submitted here: https://community.temporal.io/c/feature-requests/6

**Review by cullywakelin — APPROVED** [review]
1 NIT - otherwise, awesome

**Pull request merged**